deployment/network-operator/charts/sriov-network-operator/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: sriov-network-operator
+description: |
+  SR-IOV network operator configures and manages SR-IOV networks in the kubernetes cluster
+
+# A chart can be either an 'application' or a 'library' chart.
```

Comment:
Non-related comments

Reply:
These comments are auto generated by helm when it creates a package template, from my POV these can be left as is